public/src/components/drawer.tsx

```diff
@@ -112,6 +115,11 @@ export default function NavDrawer(): React.ReactElement {
     setState({ ...state, [anchor]: open });
   };
 
+  const isHalloween = () => {
+    const now = new Date();
+    return now.getMonth() == 9 && now.getDate() == 31;
```

Comment:
Month being 0 indexed gets me every time!

Reply:
haha I've seen so many bugs caused by this, it's horrible!